Sources/Authentication/LoginSessionConfiguration.swift

```diff
@@ -32,15 +33,17 @@ public struct LoginSessionConfiguration {
     }
     
     public init(authorizationEndpoint: URL,
-                responseType: ResponseType,
-                scopes: [Scope],
+                tokenEndpoint: URL,
+                responseType: ResponseType = .code,
+                scopes: [Scope] = [.openid, .email, .phone, .offline_access],
                 clientID: String,
-                prefersEphemeralWebSession: Bool,
+                prefersEphemeralWebSession: Bool = false,
```

Comment:
should this one be true as discussed on Slack? 👀

Reply:
yes it should 🙈